src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -392,6 +400,9 @@ public static Healthcheck create(
       if (retries != null) {
         builder.retries(retries);
       }
+      if (startPeriod != null) {
```

Comment:
I added a commit to this PR here to remove these checks. https://github.com/spotify/docker-client/pull/807